service/security/registration/simple/src/main/java/org/eclipse/kapua/security/registration/simple/SimpleRegistrationProcessor.java

```diff
@@ -273,32 +277,16 @@ private User createUser(final String name, final String email, final String disp
         final Set<Permission> permissions = new HashSet<>();
         permissions.add(permissionFactory.newPermission(new AccessInfoDomain(), Actions.read, user.getScopeId()));
 
-        permissions.add(permissionFactory.newPermission(new AccountDomain(), Actions.read, user.getScopeId()));
-
-        permissions.add(permissionFactory.newPermission(new CredentialDomain(), Actions.delete, user.getScopeId()));
-        permissions.add(permissionFactory.newPermission(new CredentialDomain(), Actions.read, user.getScopeId()));
-        permissions.add(permissionFactory.newPermission(new CredentialDomain(), Actions.write, user.getScopeId()));
-
-        permissions.add(permissionFactory.newPermission(new DatastoreDomain(), Actions.read, user.getScopeId()));
-
-        permissions.add(permissionFactory.newPermission(DeviceDomain.INSTANCE, Actions.read, user.getScopeId()));
-        permissions.add(permissionFactory.newPermission(DeviceDomain.INSTANCE, Actions.write, user.getScopeId()));
-        permissions.add(permissionFactory.newPermission(DeviceDomain.INSTANCE, Actions.delete, user.getScopeId()));
-
-        permissions.add(permissionFactory.newPermission(new DeviceConnectionDomain(), Actions.read, user.getScopeId()));
-
-        permissions.add(permissionFactory.newPermission(new DeviceEventDomain(), Actions.read, user.getScopeId()));
-        permissions.add(permissionFactory.newPermission(new DeviceEventDomain(), Actions.write, user.getScopeId()));
-
-        permissions.add(permissionFactory.newPermission(new DeviceManagementDomain(), Actions.read, user.getScopeId()));
-        permissions.add(permissionFactory.newPermission(new DeviceManagementDomain(), Actions.write, user.getScopeId()));
-        permissions.add(permissionFactory.newPermission(new DeviceManagementDomain(), Actions.execute, user.getScopeId()));
-
-        permissions.add(permissionFactory.newPermission(new GroupDomain(), Actions.read, user.getScopeId()));
-
-        permissions.add(permissionFactory.newPermission(new RoleDomain(), Actions.read, user.getScopeId()));
-
-        permissions.add(permissionFactory.newPermission(new UserDomain(), Actions.read, user.getScopeId()));
+        permissions.addAll(permissionFactory.newPermissions(new AccountDomain(), user.getScopeId(), read));
```

Comment:
We want to restrict resources of the auto-registered account? Just wondering about if would be nice to give full access to all resources of the platform.